controllers/controllers.js

```diff
@@ -10,4 +12,18 @@ exports.getCategories = (request, response, next)=>{
         .catch((error)=>{
             next(error)
         })
+}
+
+exports.getReviewById = (request, response, next) =>{
+
+    const {review_id} = request.params 
+    selectReviewById(review_id)
+        .then((review)=>{
+            console.log("in here")
```

Comment:
There are changes to make still but all console logs (except a 500 error handler one) should be removed before a final PR review